static/tag_multiselect.js

```diff
@@ -0,0 +1,27 @@
+var urlData = document.getElementById('tag-multiselect');
+const linkTagUrl = urlData.getAttribute("data-linkTagUrl");
+const unlinkTagUrl = urlData.getAttribute("data-unlinkTagUrl");
+
+var addTag = function(name, objectInfo) {
+  return function() {
+    let postData = {swap: 'none', values: {"tag_name": arguments[0], "object_info": objectInfo}};
+    htmx.ajax('POST', linkTagUrl, postData);
+  };
+};
+
+var removeTag = function(name, objectInfo) {
+  return function() {
+    let postData = {swap: 'none', values: {"tag_name": arguments[0], "object_info": objectInfo}};
+    htmx.ajax('POST', unlinkTagUrl, postData);
```

Comment:
Is there a good reason to use htmx rather than a vanilla `fetch` if we're not doing any swapping?

Reply:
Not much except that htmx handles the csrf token for us it seems. When using fetch, I need to manually get the csrf token and inject it into the request headers